.ci/bump-release-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name" | sed 's/checkstyle-//' | sed 's/"//' | sed 's/"//')
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+SET_VERSION="$LATEST_RELEASE_TAG-SNAPSHOT"
+echo SET_VERSION="$SET_VERSION"
+
+echo "bump version in pom.xml"
+mvn -e --no-transfer-progress versions:set -DgroupId=com.puppycrawl.tools -DartifactId=checkstyle \
+  -DoldVersion="${CS_RELEASE_VERSION}" -DnewVersion="${SET_VERSION}"
```

Comment:
all that you need to do (`$1` - take value from script parameter):
```
mvn -e --no-transfer-progress versions:set -DnewVersion="$1" && mvn versions:commit
```

Reply:
Changes Done